update docs (#23)
pi-top/pi-top-4-.NET-Core-API: src/Examples/PiTop.Interactive.Rover/readme.md

> This app will bootsrtap a rover app that embeds .NET Interactive. The kernels are exposed both as stdio and http interface.

Reviewer: ```suggestion
This app will bootstrap a rover app that embeds .NET Interactive. The kernels are exposed both as stdio and http interface.
```

## Setting up your pi-top

To run the app the configuration of the rover is

*  right encoder motor on port `M1`
*  left encoder motor on port `M2`
*  tilt servo on port `S2`
*  pan servo on port `S1`
*  front green leds on ports `D3` and `D4`

> *  back red leds on port `D0` and ``D5`

Reviewer: ```suggestion
*  back red leds on port `D0` and `D5`
```

*  front ultrasound on port `D7`
*  back ultrasound on port `D6`
*  sound sensor on port `A3`
*  camera on plate usb port

> ## Streaming Camera depedencies

Reviewer: ```suggestion
## Streaming Camera dependencies
```

> The rover app uses a special camera that streams the video over http, this has a dependency on the  [mjpg-streamer](https://github.com/jacksonliam/mjpg-streamer) project. Follow the instruction to isntalle the tool before running the app.

Reviewer: ```suggestion
The rover app uses a special camera that streams the video over HTTP. This has a dependency on the [mjpg-streamer](https://github.com/jacksonliam/mjpg-streamer) project. Follow the install instructions before running.
```

Author: done
